django_freeradius/base/models.py

```diff
@@ -260,6 +260,14 @@ class Meta:
     def __str__(self):
         return self.unique_id
 
+    def acct_status_type(self):
```

Comment:
if this is needed for DRF only I'm sure we can avoid modifying the model and add the necessary logic directly in the DRF serializer, see [SerializerMethodField](http://www.django-rest-framework.org/api-guide/fields/#serializermethodfield) in the DRF docs